dynawo/sources/Solvers/SolverSubModel/DYNSolverSubModel.cpp

```diff
@@ -217,14 +217,14 @@ SolverSubModel::evalFInit_KIN(N_Vector yy, N_Vector rr, void *data) {
     }
   }
 
-  std::sort(fErr.begin(), fErr.end(), mapcompabs());
+  std::sort(fErr.begin(), fErr.end(), SolverCommon::mapcompabs());
 
   if (fErr.size() > 0) {
     vector<std::pair<double, int> >::iterator it;
     int i = 0;
     for (it = fErr.begin(); it != fErr.end(); ++it) {
       Trace::debug("MODELER") << DYNLog(SolveParametersError, tolerance, it->second, irr[it->second],
-                                 subModel->getFequationByLocalIndex(it->second)) << Trace::endline;
+                                 "" /*FIXME: subModel->getFequationByLocalIndex(it->second)*/) << Trace::endline;
```

Comment:
why this ?

Reply:
If I remember well it was to remove the dependency between models-common and solvers

Reply:
I open a ticket